# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dvbd/hotfix.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- dvbd-0.7.7/src/dvbcat.cpp.vanilla	2020-05-20 17:05:07.862211904 +0200
+++ dvbd-0.7.7/src/dvbcat.cpp	2020-05-20 17:05:44.806211922 +0200
@@ -31,6 +31,7 @@
 #include <cerrno>
 #include <fstream>
 #include <fcntl.h>
+#include <unistd.h>
 
 void usage(const char *progname);
 time_t convertTime(const char *theTime);
--- dvbd-0.7.7/src/unixserversocket.cpp.vanilla	2020-05-20 17:06:16.662211938 +0200
+++ dvbd-0.7.7/src/unixserversocket.cpp	2020-05-20 17:07:09.406211963 +0200
@@ -27,6 +27,7 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <sys/un.h>
+#include <sys/stat.h>
 #include <fcntl.h>
 #include <pwd.h>
 #include <grp.h>
